docs/serverdev/index.rst

```diff
@@ -0,0 +1,130 @@
+Contributing to Sentry
+======================
+
+This part of the documentation guides you towards contributing to `Sentry <https://www.github.com/getsentry/sentry>`_.
+It covers the installation and configuration of required dependencies, as well as
+standard procedures of contribution.
+
+Setting up Your Development Environment
+---------------------------------------
+
+Macintosh OSX
+'''''''''''''
+
+Third Party Services
+~~~~~~~~~~~~~~~~~~~~
+To run the basic development server, you need to install both Redis and Postgresql.
+Below are basic commands to install them on OSX. By now, you hopefully have Homebrew installed.
+If you don't run::
+
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
+1

If you decide to keep it, you want "If you don't, run" because otherwise it reads like this is only for people who aren't runners.